src/plugins/ni/CMakeLists.txt

```diff
@@ -4,10 +4,6 @@ set (NI ${CMAKE_CURRENT_SOURCE_DIR}/nickel-1.1.0)
 
 file (GLOB NI_FILES ${NI}/src/*.c ${NI}/src/*.h)
 
-if (DEPENDENCY_PHASE AND ENABLE_ASAN)
-	remove_plugin (ni "ni reports runtime errors about unsigned integer overflows if you compile it using AddressSanitizer")
```

Comment:
Just out of interest, why does it suddenly work to include this plugin?

Reply:
plugin always compiled. plugintests always passed. but the validation tutorial did throw errors on spec mount.

we disabled the whole plugin to be on the safe side as we did not notice that we actually needed it.